.packit.yaml

```diff
@@ -27,6 +31,13 @@ packages:
     specfile_path: python-copr-common.spec
     files_to_sync:
       - python-copr-common.spec
+    actions:
+      create-archive:
+        - bash -c "tito build --tgz --test -o ."
+        - bash -c "ls -1t ./*.tar.gz | head -n 1"
+      get-current-version:
+        - bash -c "git describe --match python-copr-common-[0-9]* --abbrev=0 HEAD | egrep -o
+          [0-9]+\.[0-9]+"
```

Comment:
Unrelated to your comments, there is another way to get the current version. Not sure if you find it helpful or not.

```
[jkadlcik@zeratul cli]$ cat ../.tito/packages/python-copr-common |egrep -o [0-9]+\.[0-9]+
0.20
```

Also, `egrep` is deprecated, so we should use `grep -E` instead.